apps/judicial-system/backend/src/app/modules/file/file.service.ts

```diff
@@ -209,7 +210,9 @@ export class FileService {
               date: theCase.rulingDate,
             },
             pdf,
-            CaseFileCategory.RULING,
+            file.category === CaseFileCategory.RULING
+              ? CaseFileCategory.RULING
+              : CaseFileCategory.COURT_RECORD,
```

Comment:
This can be simplified to be just `file.category`, no need for the condition.

Reply:
Thanks 🙏